testnets/lumenxtestnet/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema":"../assetlist.schema.json",
+    "chain_name":"lumenx",
```

Comment:
this chain_name must be 'lumenxtestnet'

Reply:
Ok, I will repair it in a minute

Reply:
I see you updated the chian.json's chain_name, but I think you missed this assetlist file's chain_name property, which also must be 'lumenxtestnet'.

Reply:
Yes, unfortunatelly, sorry for the inconvenience.